pkg/helm/handlers/handler_test.go

```diff
@@ -290,7 +330,202 @@ func TestHelmHandlers_HandleGetChart(t *testing.T) {
 			if response.Body.String() != tt.expectedResponse {
 				t.Errorf("response body not matching expected is %s and received is %s", tt.expectedResponse, response.Body.String())
 			}
+		})
+	}
+}
+
+func TestHelmHandlers_HandleGetReleaseHistory(t *testing.T) {
+	tests := []struct {
+		name                string
+		expectedResponse    string
+		history             []*release.Release
+		expectedContentType string
+		error
+		httpStatusCode int
+	}{
+		{
+			name:                "Error occurred",
+			error:               errors.New("Chart path is invalid"),
+			expectedResponse:    `{"error":"Failed to list helm release history: Chart path is invalid"}`,
+			httpStatusCode:      http.StatusBadGateway,
```

Comment:
should not be "bad request"? btw, please update the error message: chart path is not used anywhere in release history request.

Reply:
keeping it BadGateway here since there are no changes with api